Paper: Modernizing computing by structural biologists with Jupyter and Colab

MooersLab opened the pull request:
If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.


*Editor*: <!--editor-->@stargaser<!--end-editor-->

*Reviewers*: <!--reviewers-list-->@stargaser<!--end-reviewers-list-->

MooersLab committed:
Numerous changes to the Introduciton.

MooersLab committed:
Merge branch '2021' of https://github.com/BlaineMooersLab/scipy_proceedings into 2021
I fell behind.

scoobies commented:
Hi!, I'm @scoobies, a friendly bot.

Type ```@scoobies help``` to discover how I can help you.

deniederhut commented:
@scoobies assign @stargaser as editor

scoobies commented:
Assigned! @stargaser is now the editor

deniederhut commented:
@scoobies build status

scoobies commented:
{"MooersLab-2021"=>{"data"=>{"build_output"=>"Invalid paper"}, "status"=>"fail"}}

deniederhut commented:
Okay, so I think we have two things happening here:

1. the `00` in front of the .rst file is tricking scoobies into thinking your paper is one of the example papers
2. the build server logs contain `File figure1.png not found` -- did you remember to commit your figures with your paper?

MooersLab committed:
CLN: rename paper file

MooersLab committed:
ENH: adds figure 1

MooersLab committed:
Added section on using Phenix

MooersLab committed:
Filled out methods section

MooersLab committed:
Filled out methods section

MooersLab committed:
Filled out methods section

MooersLab committed:
Revised the methods section with the help of Grammarly.

MooersLab committed:
Applied Grammarly to the Phenix section of the Results. Added a paragraph on the plus version of the snippet libraries to the methods.

MooersLab committed:
Added three images.

MooersLab committed:
Added newElyraSnip.png.

MooersLab committed:
Added edits to results.

MooersLab committed:
Added more information about Colab to Introduction.

MooersLab committed:
Added text to Discussion and Methods.

MooersLab committed:
Added ColabPyMOL.png and ColabT4L.png.

MooersLab committed:
Edit Colab methods section.

MooersLab committed:
Edit Colab methods section by adding several paragraphs.

deniederhut commented:
Hi @MooersLab ! Is this paper ready for review?

MooersLab commented:
It should be ready by the end of Thursday.

On Mon, Jun 7, 2021 at 6:52 PM Dillon Niederhut ***@***.***>
wrote:

> Hi @MooersLab <https://github.com/MooersLab> ! Is this paper ready for
> review?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#issuecomment-856334569>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZEC45QTVHJI7SC4WXVLTTRVLS3ANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

deniederhut commented:
Okay thanks!

MooersLab commented:
Other professional commitments have gotten in the way and prevented me from
working on the manuscript.
The manuscript will not be ready until the end of Sunday.

On Wed, Jun 9, 2021 at 6:40 AM Dillon Niederhut ***@***.***>
wrote:

> Okay thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#issuecomment-857622516>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZEC6XF2X6DMS4MRDXTQLTR5HJRANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

stargaser commented:
Thanks for the update @MooersLab, we'll look forward to seeing the updates in a couple of days.

MooersLab committed:
Added section on using Phenix.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+The user accesses the snippets through a cascading pulldown menu.
+Each snippet resides in a separate plain text file without any formatting.
+This feature dramatically eases adding new snippets by users and eases keeping the snippets under version control.
+The snippets are stored in the Jupyter data directory (which is found by entering :code:`jupyter --path`; it is in ~/Library/Jupyter/snippets).
```
`~/Library/Jupyter/snippets` must be on Mac OS?

MooersLab replied:
Modified by adding Mac OS.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+Clicking on a snippet name in the submenu triggers its insertion into the current cell in the notebook.
+The nested menu hierarchy serves well the user who is familiar with the content of the snippet libraries.
+
+Like most other snippet extnesions for Jupyter Notebook and JupyterLab, the jupyterlab-snippets externsion does not support tab stops nor tab triggers.
```
```suggestion
Like most other snippet extnesions for Jupyter Notebook and JupyterLab, the jupyterlab-snippets extension does not support tab stops or tab triggers.
```

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+The tab stops are particularly valuable because they direct the user to sites that may need changes in their parameter values and they guide the user to all of the site to ensure that none are overlooked.
+The overlooking of parameter values that require changing can be a major source of bugs.
+The tab triggers are also often mirrored, so a change at one instance of the same parameter will be propagate automatically to other identical instances of the parameter.
+To compensate for the lack of tab triggers, we inlcude a second copy of the code in the same snippet but in a comment and with the tab triggers marked with curly braces and numbers (Figure :ref:`compareSnips`).
```
typo `include`

MooersLab replied:
Thank you @stargraser. Typo is fixed.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+
+.. figure:: hbondsElyra.png 
+
+   This is yet another the caption. :label:`hbondElyra`
```
Suggest "snippets in the elyra-code-snippet extension"

MooersLab replied:
Did as suggested.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+A tag icon displays all of the available tags in the snippets as separate icons.
+The user can select tags to be used to choose snippets by clicking on the icons.
+
+Each snippet is displayed with several icons (Figure).
```
```suggestion
Each snippet is displayed with several icons (Figure :ref:`hbondElyra`).
```

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+A search box at the top of the list of snippets is used to recover a snippet (Figure :ref:`ColabPyMOL`.
+The user enters a snippet name in the search box to display the snippet and its documentation.
+The user hits the 'Install' button to install the snippet's code at the current position in the  notebook.
+Unlike the Elyra snippets which insert a whole snippet into one code block, a Colab snippet can have multiple code blocks that are inseted into the notebook at the current position of the mouse cursor.
```
typo -> `inserted`

MooersLab replied:
Corrected.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+For example, the snippet in Figure :ref:`ColabPyMOL` has a three blocks of Python code, two blocks of shell commands, and two blocks of bash cell majics with multiple lines of bash commands.
+
+The list snippet for a library will print in a table below the current cell a list of the snippets in the library and a brief description. 
+This table is stored in a panda's dataframe that can be searched with the panda's search function. 
```
```suggestion
This table is stored in a pandas dataframe that can be searched with the panda's search function. 
```

MooersLab replied:
Thank you for catching this typo! Correction made.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+This table can also be searched for key terms with the search function in the notebook.
+The code block and output can be hidden by clicking on the three blue dots on the left margin of the cell. 
+
+Notebooks on Colab open lighting fast, but the user must reinstall their software on each login.
```
```suggestion
Notebooks on Colab open very quickly, but the user must reinstall their software on each login.
```

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+Twenty minutes of applying simulated annealing to an early model that still has numerous errors can significantly improve the model while saving the user a day or more of the tedious manual rebuilding of the molecular model. 
+
+More recently, Phenix has been extended to refine crystal structures with neutron diffraction data and for structure determination and refinement with cryo-electron microscopy (cryo-EM) data.
+The addition of support for cryo-EM help address the recent need for the ability to fit atomic models to cryo-EM maps that have recently become available at near atomic resolution because of the dramatic imprvoements in detector technology []_.
```
typo  -> `improvements`, and need to complete the reference.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+
+Jupyter Lab and its extensions are also best run with Python3.
+The most practical approach to using Phenix in Jupyter Lab is to invoke Phenix by utilizing the shell rather than using Python.
+For example, the command shown below invokes statistical analysis of the B-factors in a Protein Data Bank (PDB) file by using one line of code in the shell (Figure :figure:XXXXXX).
```
I think the figure is not needed; just move the bash code-block closer to this paragraph.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+.. table:: Table of libraries. :label:`mtable`
+
+   +--------------------+-----------------------------------+
+   | Programs           | url                               |
```
URLs are incomplete.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+
+
+
+JupyterLab snippets
```
several empty sections headings and two figures should be removed.

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+
+.. table:: Table of tutorial jupyter notebooks. :label:`jnbtable`
+
+   +------------------------+-----------------------------------+
```
notebook names are empty

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+
+The libraries have to be molecular graphics program specific because molecular graphics programs have been written in a range of programming languages.
+The user issues the commands in either in a general programming language like Python or a a domain specific language (DSL) like pml.
+To counter this growing tower of babel, the XXXXXX intiative was formed cite. 
```
What is the initiative?

stargaser commented on papers/blaine_mooers/mooersblaine.rst:
```diff
@@ -0,0 +1,709 @@
+           doi: 10.1107/S1600576720013412
+           
+.. [Mott10] S.E. Mottarella, M. Rosa, A. Bangura, H.J. Bernstein, and P.A. Craig.
+            *Conscript: RasMol to PyMOL script converter*,
```
the extra space seems to cause a problem when building the paper.

MooersLab replied:
Corrected. Solved problem with compiling.

stargaser reviewed (changes requested):
@MooersLab although this paper is not quite complete, the proceedings committee decided it is close enough to make review comments which may help you to complete it.

My domain of expertise is astronomy so I have little knowledge of structural biology. Nonetheless, the demonstration of code snippets to guide users is very interesting to learn about. I have not encountered these in my work so far. 

The main thing to address is the incomplete parts that really start to show up on p. 6. I have pointed these out in the in-line comments. Table 2 and Table 3 have missing column data and Figs 6 and 7 are placeholders. Several section headings appear on p. 6.

The narrative in the paper moves along pretty well in my opinion through the first four pages. Phenix is given a lot of attention on p. 5 which leads me to believe that Phenix might be more  important to structural biologists than the other packages? Or is the extra attention due to running Phenix via shell instead of through notebooks? 

References -- I appreciate that you've provided dois for these. 

I feel there should be some kind of url reference at least for Colab, if there is not a formal citation.

I encourage you to complete the missing parts.

stargaser commented:
@scoobies add @stargaser to reviewers

scoobies commented:
@stargaser added to the reviewers list!

stargaser commented:
@scoobie mark needs more review

stargaser commented:
@scoobies mark needs more review

MooersLab commented:
Thank you very much @stargaser for the feedback and recommended changes!!!

stargaser commented:
@scoobies mark pending comment

stargaser commented:
@MooersLab please update by July 1st to make it in time for the conference-ready proceedings

MooersLab commented:
Will do!

On Sat, Jun 26, 2021 at 2:11 PM David Shupe ***@***.***>
wrote:

> @MooersLab <https://github.com/MooersLab> please update by July 1st to
> make it in time for the conference-ready proceedings
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#issuecomment-869048759>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZEC25BAITVZDWX6NVAETTUYQ4JANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

MooersLab committed:
Updated the text.

MooersLab committed:
Update papers/blaine_mooers/mooersblaine.rst

Co-authored-by: David Shupe <shupe@ipac.caltech.edu>

MooersLab committed:
Update papers/blaine_mooers/mooersblaine.rst

Co-authored-by: David Shupe <shupe@ipac.caltech.edu>

MooersLab committed:
Updated the text.

MooersLab committed:
Resolved merge conflict.

MooersLab committed:
Updated the text.

MooersLab committed:
Updated the text.

MooersLab committed:
Updated the text.

MooersLab committed:
Minor edits.

MooersLab committed:
Minor edits.

MooersLab committed:
Minor edits.

MooersLab commented:
Hi David,

I have corrected, truncated, and updated the manuscript for the
conference-ready proceedings.
It can stand alone for now.
I will make more additions later.
I am tied up for the next 36 hour with service on a NIH Study Section.

Best regards,

Blaine


On Sat, Jun 26, 2021 at 2:11 PM David Shupe ***@***.***>
wrote:

> @MooersLab <https://github.com/MooersLab> please update by July 1st to
> make it in time for the conference-ready proceedings
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#issuecomment-869048759>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZEC25BAITVZDWX6NVAETTUYQ4JANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

stargaser commented:
@MooersLab thank you for the revised version. Some items needing attention:

Figure 3 caption

References for Jupyter, and other s/w packages used. These are on the scipy.org site. 

I'm afraid I'm away from my computer and I'm typing this from memory from reading the paper this morning. The proceedings committee expects this paper may slip to the post-conference publication, unless these are items to be addressed in the next day.

MooersLab commented:
I will fix address those issues in the next day. I finally done with the
NIH Study Section.

On Fri, Jul 2, 2021 at 4:03 PM David Shupe ***@***.***> wrote:

> @MooersLab <https://github.com/MooersLab> thank you for the revised
> version. Some items needing attention:
>
> Figure 3 caption
>
> References for Jupyter, and other s/w packages used. These are on the
> scipy.org site.
>
> I'm afraid I'm away from my computer and I'm typing this from memory from
> reading the paper this morning. The proceedings committee expects this
> paper may slip to the post-conference publication, unless these are items
> to be addressed in the next day.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#issuecomment-873254974>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZEC6QU6Y6EJE3W35MLWTTVYSQ3ANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

stargaser commented:
Glad to hear you made it through the NIH Study Section @MooersLab .

I believe this is the Jupyter paper that @deniederhut let me know about:
https://doi.org/10.1109/MCSE.2021.3059263 

I think with that and filling out the caption for Figure 3, that would get the paper over the finish line.

MooersLab commented:
Hi David,

Sorry about the empty message. I forgot to save my text while using
firenvim in gmail.

I made the suggested changes to the manuscript and ran `git push`.

Blaine



On Fri, Jul 2, 2021 at 9:41 PM David Shupe ***@***.***> wrote:

> Glad to hear you made it through the NIH Study Section @MooersLab
> <https://github.com/MooersLab> .
>
> I believe this is the Jupyter paper that @deniederhut
> <https://github.com/deniederhut> let me know about:
> https://doi.org/10.1109/MCSE.2021.3059263
>
> I think with that and filling out the caption for Figure 3, that would get
> the paper over the finish line.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#issuecomment-873331888>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZECYHWL3IXOY2GEH3BFDTVZ2GFANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

stargaser commented:
Hi Blaine,

I don't see a new commit in this pull request and the PDF looks unchanged. Would you please try pushing the changes again?

David

MooersLab committed:
Made recommended edits and added several references.

MooersLab commented:
Hi David,

I am sorry about that!  I forgot to add and commit my changes yesterday.
I have now done so, and I pushed my changes a few minutes ago.

Blaine

On Sun, Jul 4, 2021 at 11:47 AM David Shupe ***@***.***>
wrote:

> Hi Blaine,
>
> I don't see a new commit in this pull request and the PDF looks unchanged.
> Would you please try pushing the changes again?
>
> David
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#issuecomment-873624052>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZECYEUMRRPON5SXQUJ2LTWCGDBANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

MooersLab committed:
Added a paragraph to the results, added another figure, and converted several figures to two column wide to enlargen the text in the images.

stargaser reviewed (approved):
These revisions look good to me. Thanks, Blaine!

stargaser commented:
@scoobies mark ready

deniederhut commented on AAAreadme.md:
```diff
@@ -0,0 +1,52 @@
+<h1> AAAreadme file </h1>
```
Did you mean to commit this?

deniederhut commented on Untitled.ipynb:
```diff
@@ -0,0 +1,33 @@
+{
```
Same question here - did you mean to commit this?

deniederhut commented on papers/blaine_mooers/scipy21bib.bib:
```diff
@@ -0,0 +1,8 @@
+@Book{hume48,
```
Are you using this bibfile in your paper?

deniederhut commented on update.sh:
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```
Did you mean to commit this as well?

MooersLab commented:
No, I added the citations manually.

On Mon, Jul 12, 2021 at 7:57 PM Dillon Niederhut ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In papers/blaine_mooers/scipy21bib.bib
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#discussion_r668351701>
> :
>
> > @@ -0,0 +1,8 @@
> +@Book{hume48,
>
> Are you using this bibfile in your paper?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#pullrequestreview-704656422>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZEC6L5JUWSC433WS2NC3TXOFPXANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

MooersLab commented:
No, update.sh was included by accident.

On Mon, Jul 12, 2021 at 7:57 PM Dillon Niederhut ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In update.sh
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#discussion_r668351863>
> :
>
> > @@ -0,0 +1,2 @@
> +#!/bin/bash
>
> Did you mean to commit this as well?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#pullrequestreview-704656578>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZEC64JOX2Y2QFRJL6TMLTXOFRPANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

deniederhut commented:
Okay thanks for clarifying. Would it make sense to remove the extraneous files?

MooersLab committed:
remove update.sh

MooersLab committed:
removed papers/blaine_mooers/scipy21bib.bib and papers/blaine_mooers/scipy21bib2.bib'

MooersLab commented:
Yes, I just did so.

On Wed, Jul 14, 2021 at 8:52 PM Dillon Niederhut ***@***.***>
wrote:

> Okay thanks for clarifying. Would it make sense to remove the extraneous
> files?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/635#issuecomment-880327053>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTZEC4HUUGRO6ZA67UWYPTTXY5PLANCNFSM45X2JHLA>
> .
>


-- 
Best regards,

Blaine

Blaine Mooers, Ph.D.
Associate Professor
Department of Biochemistry and Molecular Biology
College of Medicine
University of Oklahoma Health Sciences Center
S.L. Young Biomedical Research Center Rm. 466
975 NE 10th Street
<https://maps.google.com/?q=975+NE+10th+Street&entry=gmail&source=g>, BRC
466
Oklahoma City, OK 73104-5419

office: (405) 271-8300   lab: (405) 271-8313

Faculty webpage
<http://basicsciences.ouhsc.edu/biochemmolbiol/Faculty/bio_details/TabId/11753/ArtMID/30702/ArticleID/6430/Mooers-Blaine-HM-PhD.aspx>
   X-ray lab (LBSF)
<http://research.ouhsc.edu/CoreFacilities/LaboratoryofBiomolecularStructureandFunction.aspx>
SSRL
UEC
<https://www-ssrl.slac.stanford.edu/content/about-ssrl/advisory-panels/ssrl-users-organization/members/ssrluo-2016-executive-committee-members>
SSURF
EasyPyMOL <https://github.com/MooersLab/EasyPyMOL> Molecular Graphics
<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/moleculargraphicslinks.html>

<https://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/MolecularGraphicsLinks.html>
Small
Angle Scattering
<http://www.oumedicine.com/docs/default-source/ad-biochemistry-workfiles/small-angle-scattering-links-27aug2014.html?sfvrsn=0>
office: (405) 271-8300   lab: (405) 271-8313
e-mail:  ***@***.*** (or ***@***.***)

deniederhut commented:
Okay thanks!
